Nodejs/Product/Npm/SPI/Person.cs

```diff
@@ -71,9 +83,9 @@ internal class Person : IPerson {
             // Verify we are parsing correctly
             try {
                 var jObject = JObject.Parse(source);
-                Debug.Assert(((string)jObject["name"] ?? string.Empty) == Name, string.Format("Failed to parse name from {0}", source));
-                Debug.Assert(((string)jObject["email"] ?? string.Empty) == Email, string.Format("Failed to parse email from {0}", source));
-                Debug.Assert(((string)jObject["url"] ?? string.Empty) == Url, string.Format("Failed to parse url from {0}", source));
+                Debug.Assert((string)jObject["name"] == Name, string.Format("Failed to parse name from {0}", source));

```

Comment:
Why bother with this... we're extracting them using the regex, do we still care what the json parser makes of it...